saleor/graphql/payment/mutations.py

```diff
@@ -160,6 +160,15 @@ def validate_return_url(cls, input_data):
                 {"redirect_url": error}, code=PaymentErrorCode.INVALID
             )
 
+    @classmethod
+    def check_covered_amount(cls, checkout, checkout_total, amount):
```

Comment:
Can we add typing here? Example `checkout_total`  doesn't tell me if I should pass the value as a `Money` or `TaxedMoney`.